winbuild/Makefile.vc

```diff
@@ -21,10 +22,20 @@ CFGSET=true
 !MESSAGE                                  Libraries can be fetched at http://pecl2.php.net/downloads/php-windows-builds/
 !MESSAGE                                  Uncompress them into the deps folder.
 !MESSAGE   WITH_SSL=<dll or static>     - Enable OpenSSL support, DLL or static
+!MESSAGE      SSL_LIBS=<File1 File2>    - OpenSSL libs to use (default: libeay32.lib ssleay32.lib) (requires WITH_SSL)
+!MESSAGE      SSL_HEADER_PATH=<path>    - Path to OpenSSL include files. (requires WITH_SSL)
```

Comment:
I'm planning to rebase ASAP, it might be the weekend however.  Sorry

Names are just names.  I'm happy to take guidance and change the patch to suite.  In the interest of patch size I chose the names used inside the slave makefile.